configs/local-config.json

```diff
@@ -13,6 +13,7 @@
   "l_votingPeriod": 300,
   "m_votingDelay": 300,
   "n_quorumPercentage": 10,
-  "o_enableGovernaceToken": true,
-  "p_enableMembershipToken": true
-}
```

Comment:
Do we need to remove line 15 and add 
`n_quorumNumerator` here like in the example.json?

Reply:
Thanks! I have that change locally, but because it's ignored, it didn't pick it up. I'll get that fixed.